bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/i18n/unifi.properties

```
@@ -164,3 +177,16 @@ error.thing.offline.configuration_error = You must choose a UniFi Controller for
 error.thing.poe.offline.configuration_error = The configuration parameter macAddress must be set and not be empty.
 error.thing.poe.offline.nodata_error = No data for the PoE port could be found in the UniFi API data. See TRACE log for actual API data.
 error.thing.site.offline.configuration_error = The configuration parameter sid must be set and not be empty.
+
+# actions
+
+generateVouchersLabel = Generate
```

I would suggest to use a convention similar to the other keys:
```suggestion
actions.unifi.generateVouchers.label = Generate
```

---

Done